### PR TITLE
Add developer set-up notes

### PR DESCRIPTION
**Pull request by illusional**
@danielreti, can you add any additional information that you found when setting up seqr for local development.

### CODE REVIEW

**Comment by danielreti** [comment]
I think set up the env before the migrations and also make sure google cloud is either in the env or globally available on the terminal.

**Comment by illusional** [comment]
Thanks @danielreti, @daniaki, if you haven't set-up the seqr environment yet, can you follow these instructions and leave any feedback? Thanks!

File: developer.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+   npm install
+   npm run build
+   cp dist/* ../static/
+   cd ..
+   ```
+
+1. Build seqr python environment
+
+   ```shell
+   conda env create -n seqr python=3.7
```
Comment (daniaki) [inline developer.md:29]: Apparently `conda env create` no longer works and it should `conda create`. 

https://github.com/conda/conda/issues/3859

Comment (illusional) [reply developer.md]: Thanks @daniaki!

**Review by daniaki — APPROVED** [review]

**Pull request merged**